extensions/amp-story-interactive/amp-story-interactive.md

```diff
@@ -1,10 +1,308 @@
+You may obtain a copy of the License at
 
-- Quiz: Contains objective correct answers that users try to guess (2-4)
-- Poll: Contains subjective questions and options that users vote (2-4)
-- Binary poll: Subjective poll with only two options and a more engaging layout
-- Results page: Aggregate results from the user inputs on the previous interactive components of a story.
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+[tip type="important"]Change image sources to ampproject/amphtml/master!!![/tip]
+
+The amp-story-interactive component provides a set of experiences, such as quizzes or polls, for users in [Web stories](https://amp.dev/documentation/guides-and-tutorials/start/create_successful_stories/?format=stories). Interactive experiences provided by amp-story-interactive can integrate into tools or editors and support analytics.
+
+<div layout="container" width="3" height="2">
+<div style="width:32%;display:inline-block"><amp-img src="https://github.com/mszylkowski/amphtml/raw/interactive_docs/extensions/amp-story-interactive/img/quiz-art.png" layout="responsive" width="200" height="350"/></div>
+<div style="width:32%;display:inline-block"><amp-img src="https://github.com/mszylkowski/amphtml/raw/interactive_docs/extensions/amp-story-interactive/img/animal-poll.png" layout="responsive" width="200" height="350"/></div>
+<div style="width:32%;display:inline-block"><amp-img src="https://github.com/mszylkowski/amphtml/raw/interactive_docs/extensions/amp-story-interactive/img/results-art.png" layout="responsive" width="200" height="350"/></div>
+</div>
+
+## Usage
+
+The amp-story-interactive component encompasses a set of interactive experiences. Specify an interactive experience by defining one of the elements below. For best results, only use one element per amp-story-page.
+
+If you want to see all the components in action, check out the [example story](https://amp.dev/documentation/examples/components/amp-story-interactive-poll/story#page=title-components)
+
+### amp-story-interactive-binary-poll
+
+The `amp-story-interactive-binary-poll` element provides a two option voting user interface. Users may select one of two valid options. When selected, the highlighted option fills the container and displays the total percentage of votes.
+
+Does not support pairing with `amp-story-interactive-results`, and can optionally have a prompt.
+
+<amp-img src="https://github.com/mszylkowski/amphtml/raw/interactive_docs/extensions/amp-story-interactive/img/binary-poll-raw.png" layout="intrinsic" width="400" height="230">
+
+```html
+<amp-story-interactive-binary-poll
+  id="pizza-binary-poll"
+  endpoint="https://endpoint.com/v1/interactives"
```

Comment:
This needs to ship with the Google backend as the default endpoint

Reply:
Can't include it here yet since it's not available. When it becomes available I plan to update these examples. Notice the "coming soon" on the backend section 😁

Reply:
I think we should wait until it's ready before submitting this documentation!

Reply:
Independently, having the docs approved is a good way of getting ready for launching!

Reply:
It supports any backends that conform to the API described in the "Aggregate data source" section. Anyone can make a backend with that API and plug the endpoint here. Google will provide an endpoint for convenience, so we will add the endpoint to the docs once we have everything ready.